### PR TITLE
added MessageFormat support

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

File: pkg/convert/convert.go [hunk 1]
```diff
@@ -8,3 +10,15 @@ func convertToMessageFormatSingular(message string) string {
 
 	return "{" + message + "}"
 }
+
+// convertFromMessageFormatStrToStr function replaces '{' and '}' only if message starts with '{' and ends with '}'.
```
Comment (klavinsjanis) [inline pkg/convert/convert.go:14]: This seems like it could be a generic function e.g. -
removeBracketsFromString()

File: pkg/convert/convert.go [hunk 2]
```diff
@@ -8,3 +10,15 @@ func convertToMessageFormatSingular(message string) string {
...
+func convertFromMessageFormatStrToStr(message string) string {
+	if strings.HasPrefix(message, "{") && strings.HasSuffix(message, "}") {
```
Comment (janishorsts) [inline pkg/convert/convert.go:16]: The value stored in `Message` field is ALWAYS encoded using MessageFormat. Is this function temporary?

File: pkg/convert/convert.go [hunk 3]
```diff
@@ -8,3 +10,15 @@ func convertToMessageFormatSingular(message string) string {
...
+		str := strings.TrimSuffix(message, "}")
+		str = strings.TrimPrefix(str, "{")
```
Comment (janishorsts) [inline pkg/convert/convert.go:18]:
```suggestion
		return str[1:len(str)-1]
```

File: pkg/convert/convert.go [hunk 4]
```diff
@@ -8,3 +10,9 @@ func convertToMessageFormatSingular(message string) string {
 
 	return "{" + message + "}"
 }
+
+// removeEnclosingBrackets replaces '{' and '}', temporarily maintain only the singular form.
+func removeEnclosingBrackets(message string) string {
+	replacer := strings.NewReplacer("{", "", "}", "")
```
[the remark this answers is not in the document]
Comment (klavinsjanis) [reply pkg/convert/convert.go:16]: Won't this replace all occurrences of curly braces?

[the remark this answers is not in the document]
Comment (klavinsjanis) [reply pkg/convert/convert.go:16]: might be worth adding more test cases

[the remark this answers is not in the document]
Comment (DarjaLeonova) [reply pkg/convert/convert.go:16]: I had a call with Janis and he said that this function can be used as a temporary solution.

[the remark this answers is not in the document]
Comment (janishorsts) [reply pkg/convert/convert.go:16]: This is temporary, short-lived, we'll make it right in the foreseeable future. In reality, we require to parse the value formatted as MessageFormat and transform it to the target value if possible.

@DarjaLeonova can you please add a comment like `// TODO(DarjaLeonova): requires full rewrite by using MessageFormat parsing.`?

**Review by janishorsts — APPROVED** [review]

**Pull request merged**